LibreOfficeWriter_Char.au3

```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Font.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Character Styles.
```

Comment:
styles ?

Reply:
Yes, Libre Office has two ways of applying such things as "Bold", Direct formatting and Character Styles.
https://help.libreoffice.org/7.2/en-US/text/swriter/01/05130002.html

Reply:
but here we have LibreOfficeWriter_Char.au3 not styles.... so I was confused.
But now I get it what you mean.
`Characters Styles` not `Styles`